app/guid-file/route.ts

```diff
@@ -54,7 +58,28 @@ export default class GuidFile extends Route {
                 this.intl.t('general.presented_by_osf'),
             language: this.metadata.language ? this.metadata.language : undefined,
         };
-        this.set('headTags', this.metaTags.getHeadTags(metaTagsData));
+
+        // Google Structured Data
+        const parentId = await model.target.get('id');
+        const jsonLD: object = await this.scriptTags.returnStructuredData(parentId);
+        const jsonString: string = Object.entries(jsonLD) ?
+            JSON.stringify(jsonLD) : JSON.stringify({ isAccessibleForFree : true });
```

Comment:
This key/value pair came directly from the Structured Data Dataset. We are providing a minimum dataset in this instance. I have a more complete default dataset available should we like. If you fully read about any data that would result in a manual action, it will simply not be displayed in the rich results text. This is not inherently a problem. It is an issue to not have a spam check if we are presenting pages that are spam, but this will also add to the scope of the ticket. We should speak further with the BE to know whether and how spam checks are performed and how that affects the dataset supplied.

Reply:
FE providing a default to indexers like Google sounds like a scope creep already. It defeats the purpose of retrieving JSON-LD data from the BE as we might as well just build everything on the FE.